lib/presentation/destinations/weather/home/home_page.dart

```diff
@@ -46,7 +48,22 @@ class HomePage extends ConsumerWidget {
         ),
         const ThemePicker(),
       ],
-      body: const HomePageBody(),
+      body: HomePageBody(),
+      floatingActionButton: FloatingActionButton(onPressed: ()async{
+          final ImagePicker _picker = ImagePicker();
+          final XFile? image =
+              await _picker.pickImage(source: ImageSource.gallery);
+          if (image != null && image.path.endsWith('.svg')) {
+            final svgString = await File(image.path).readAsString();
+            ref.read(svgProvider.notifier).addSVG(svgString);
```

Comment:
.addSvg is an async method and you should add await. Else the snackbar appears before it is added.